src/Adapters/IngressAdapter.ts

```diff
@@ -0,0 +1 @@
+export default class IngressAdapter {}
```

Comment:
I assume you will work on this as you make/implement adapter calls?

Reply:
yep